docs/sources/setup/install/helm/reference.md

```diff
@@ -3120,6 +3120,24 @@ null
 			<td><pre lang="json">
 null
 </pre>
+</td>
+		</tr>
```

Comment:
Did you edit this file directly? Or was this change automatically generated? The Helm reference should be updated in production/helm/loki/reference.md.gotmlp. There's a note to that effect at line 11 of the docs/sources/setup/install/helm/reference.md. file, you might have missed it.

Reply:
I didn't edit this file manually, I ran the following command : `make -C docs sources/setup/install/helm/reference.md` but I guess according to the comment you mentioned that I should run `make -C docs production/helm/loki/reference.md.gotmpl` right ?

Reply:
@QuantumEnigmaa I was trying to find out if you had manually changed `reference.md` or if the changes to that file were automatically generated.  If you ran `make -C docs sources/setup/install/helm/reference.md` then the changes were automatically generated, so everything is as it should be.   

If you had manually updated the file, the changes would have been overwritten the next time someone ran `make -C docs sources/setup/install/helm/reference.md`

Reply:
Oh ok I see. Is everything ok to merge on your side then ?

Reply:
I'm a technical writer, I only review the doc parts of the PR.  One of the engineers will have to approve the Helm charts updates.